Source/WebCore/platform/animation/TimingFunction.h

```diff
@@ -138,7 +140,9 @@ class CubicBezierTimingFunction final : public TimingFunction {
 
     static const CubicBezierTimingFunction& defaultTimingFunction()
     {
+IGNORE_GCC_WARNINGS_BEGIN("dangling-reference")
         static const CubicBezierTimingFunction& function = create().leakRef();
```

Comment:
Wouldn't it be better to use `NeverDestroyed<>` for these?

Reply:
`NeverDestroyed<Ref<LinearTimingFunction>>` works.